pkg/core/localize/locales/en/cmd/context.en.toml

```
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
 
 [context.cmd.longDescription]
 one='''
-rhoas context commands allow developers to:
+Group your application service instances into service contexts.
```

As first sentence it sounds strange to me.
Maybe shorter form?

```suggestion
Group your service instances into contexts.
```

---

In general for the CLI, I try to specify "application services" when referring to one of the RHOAS services. To distinguish them from the services that devs are building. Instead of "application services", how about "RHOAS services"?

---

We cannot use acronyms like rhoas